crates/jstz_cli/src/sandbox/daemon.rs

```diff
@@ -153,7 +168,13 @@ fn generate_identity(cfg: &Config) -> Result<()> {
     Ok(())
 }
 
-fn init_node(cfg: &Config) -> Result<()> {
+const MAX_PROGRESS: u32 = 16;
```

Comment:
Maybe a comment on how this constant was produced?

Reply:
added